gl2csv.py

```diff
@@ -37,30 +36,43 @@ def main():
     parser.add_argument('--project',
                     help='Gitlab project name (i.e. webteam/fhnw.webauftritt)',
                     default='webteam/fhnw.webauftritt')
+    parser.add_argument('--output',
+                    help='Filename to output to',
+                    default='issues.csv')
+
     args = parser.parse_args()
     gl = gitlab.Gitlab(args.url, args.token)
-    print(args.project)
-    project = gl.projects.get(id=25)
-    #issues =  project.issues.list(all=True)
```

Comment:
Please don't remove. Pickeling is a good way to save network traffic. It could also be generalized though.

Reply:
Looks like a misplaced comment?

I do remember removing the pickling code somewhere else... Will relook into it there.